risc0/circuit/keccak/Cargo.toml

```diff
@@ -34,6 +34,7 @@ risc0-circuit-keccak-methods = { path = "methods" }
 risc0-zkvm = { workspace = true, features = [
   "std",
   "unstable",
+  "prove",
```

Comment:
If we always assume `prove`, then do we still need a feature flag for this crate?

Reply:
yes - the prove flag is set for dev-dependencies - I changed the test for this crate to not use coprocessor so it exercises the pending_keccaks code path (which is similar to the pending_zkrs thing we do) in the server.